pages/domains/domains_before.md

```diff
@@ -3,33 +3,34 @@ title: Before you request a .gov domain
 permalink: /domains/before/
 layout: layouts/info-page
 sidenav: true
-excerpt: Follow these steps to complete your request as quickly as possible
+excerpt: Follow these steps to complete your request as quickly as possible.
 tags: domains
 eleventyNavigation:
   key: domains
-  order: 2
+  order: 3
   title: Before you request a .gov domain
 ---
 
 If you’re ready to request your .gov domain then let’s get started. You don’t have to complete the process in one session. You can save what you enter and come back to it when you’re ready.
 
-<button class="usa-button">Start your .gov  domain request </button>
+<button class="usa-button">Start a .gov domain request </button>
```

Comment:
I think this should stay a link element, but we can style it like a button. You can do that either by changing the button to a link element <a> or using the markdown attribute so it would be in which case it would be [Start a .gov domain request](#){.usa-button}